nvflare/job_config/api.py

```diff
@@ -572,21 +572,29 @@ def export_job(self, job_root: str):
         self.job.generate_job_config(job_root)
 
     def simulator_run(
-        self, workspace: str, n_clients: int = None, threads: int = None, gpu: str = None, log_config: str = None
+        self,
+        workspace: str,
+        n_clients: Optional[int] = None,
+        clients: Optional[List[str]] = None,
+        threads: Optional[int] = None,
+        gpu: Optional[str] = None,
+        log_config: Optional[str] = None,
     ):
         """Run the job with the simulator with the `workspace` using `clients` and `threads`.
         For end users.
 
         Args:
             workspace: workspace directory for job.
             n_clients: number of clients.
+            clients: client names.
             threads: number of threads.
             gpu: gpu assignments for simulating clients, comma separated
             log_config: log config mode ('concise', 'default', 'verbose'), filepath, or level
 
         Returns:
-
         """
+        if clients:
+            self.clients = clients
```

Comment:
Should check to make sure length of clients is consistent with n_clients (or allow n_clients to be None)